modules/dop/dao/testplan_testcase_relation.go

```diff
@@ -323,7 +324,7 @@ func (client *DBClient) PagingTestPlanCaseRelations(req apistructs.TestPlanCaseR
 	}
 	// updater
 	if len(req.UpdaterIDs) > 0 {
-		baseSQL = baseSQL.Where("`tc`.`updater_id` IN (?)", req.UpdaterIDs)
+		baseSQL = baseSQL.Where("`rel`.`updater_id` IN (?)", req.UpdaterIDs)
```

Comment:
why change tc to rel

Reply:
Both `TestPlanCaseRelation` and `TestCase` have `updater_id` field.
In method `PagingTestPlanCaseRelations`, we should use `rel`'s `updater_id`.